lua/astrocommunity/note-taking/neorg/init.lua

```diff
@@ -1,23 +1,29 @@
 return {
-  "nvim-neorg/neorg",
-  build = ":Neorg sync-parsers",
-  dependencies = { "nvim-lua/plenary.nvim" },
-  event = "VeryLazy",
-  opts = {
-    load = {
-      ["core.defaults"] = {}, -- Loads default behaviour
-      ["core.concealer"] = {}, -- Adds pretty icons to your documents
-      ["core.keybinds"] = {}, -- Adds default keybindings
-      ["core.completion"] = {
-        config = {
-          engine = "nvim-cmp",
-        },
-      }, -- Enables support for completion plugins
-      ["core.journal"] = {}, -- Enables support for the journal module
-      ["core.dirman"] = { -- Manages Neorg workspaces
-        config = {
-          workspaces = {
-            notes = "~/projects/notes",
+  {
+    "vhyrro/luarocks.nvim",
+    priority = 1000, -- We'd like this plugin to load first out of the rest
+    config = true,
+  },
```

Comment:
This should. be a dependencies of neorg.

Reply:
fixed